recipes/variantbam/meta.yaml

```diff
@@ -4,19 +4,24 @@ package:
 
 build:
   number: 0
-  skip: True # [osx]
+  skip: True
```

Comment:
The variantbam build fails with an error that seems unrelated to the pinning changes. I have added the skip in order to notify the author once this is merged.